### PR TITLE
adds target="_blank" to external links

### LINKED ISSUE #47: Open links to new tabs
add target:"_blank" to open externals links in new tabs (partner url, social networks,...)

### PR DESCRIPTION
**Pull request by jasonzubiate**
This change resolves #47 by adding target="_blank" to all external links found on the webpage.
Signed-off-by: Jason Zubiate <jasonzubiate.com>

### CODE REVIEW

File: src/components/Footer.vue [hunk 1]
```diff
@@ -8,7 +8,7 @@
         Réseaux
       </h1>
       <div class="flex w-min flex-row">
-        <a class="hover:blur-sm" href="https://www.instagram.com/insalan_officiel">
+        <a class="hover:blur-sm" href="https://www.instagram.com/insalan_officiel" target="_blank">
```
Comment (KwikKill) [inline src/components/Footer.vue:11]: There is an error raised by ESlint. You should add `rel="noopener noreferrer"` when using `target="_blank"` for security reason.

File: src/components/Footer.vue [hunk 2]
```diff
@@ -111,7 +111,7 @@
             </g>
           </svg>
         </a>
-        <a class="hover:blur-sm" href="https://twitter.com/insalan">
+        <a class="hover:blur-sm" href="https://twitter.com/insalan" target="_blank">
```
Comment (KwikKill) [inline src/components/Footer.vue:114]: missing `rel="noopener noreferrer"`.

File: src/components/Footer.vue [hunk 3]
```diff
@@ -142,7 +142,7 @@
             </g>
           </svg>
         </a>
-        <a class="hover:blur-sm" href="https://www.facebook.com/insalan/">
+        <a class="hover:blur-sm" href="https://www.facebook.com/insalan/" target="_blank">
```
Comment (KwikKill) [inline src/components/Footer.vue:145]: missing `rel="noopener noreferrer"`.

File: src/components/Footer.vue [hunk 4]
```diff
@@ -192,14 +192,14 @@
     </div>
     <div class="col-span-3 flex flex-row">
       <div class="mx-2 flex flex-row justify-end">
-        <a class="mt-2 md:mt-5 " href="https://aeir.insa-rennes.fr">
+        <a class="mt-2 md:mt-5 " href="https://aeir.insa-rennes.fr" target="_blank">
```
Comment (KwikKill) [inline src/components/Footer.vue:195]: missing `rel="noopener noreferrer"`.

File: src/components/Footer.vue [hunk 5]
```diff
@@ -192,14 +192,14 @@
...
           <img
             alt="Logo de l'AEIR"
             class="h-[32px] md:h-[64px]"
             src="@/assets/images/logo_aeir.png"
           />
         </a>
-        <a class="ml-2 mt-2 md:mt-5" href="https://insa-rennes.fr">
+        <a class="ml-2 mt-2 md:mt-5" href="https://insa-rennes.fr" target="_blank">
```
Comment (KwikKill) [inline src/components/Footer.vue:202]: missing `rel="noopener noreferrer"`.

File: src/components/PartnerCard.vue [hunk 6]
```diff
@@ -12,7 +12,7 @@ const { partner } = toRefs(props);
 
 <template>
   <div :key="partner.id" class="container h-48 break-words bg-cyan-900 shadow-lg hover:bg-cyan-800">
-    <a :href="partner.url" class="flex h-full flex-col items-center gap-2 p-2">
+    <a :href="partner.url" target="_blank" class="flex h-full flex-col items-center gap-2 p-2">
```
Comment (KwikKill) [inline src/components/PartnerCard.vue:15]: missing `rel="noopener noreferrer"`.

**Review by KwikKill — CHANGES_REQUESTED** [review]
Thank you for your time, this is good but there is a slight modification required

File: src/components/PartnerCard.vue (hunk 6 above)
Comment (jasonzubiate) [reply src/components/PartnerCard.vue:15]: will add those

**Review by KwikKill — APPROVED** [review]
Will be merged later

**Pull request closed without merging**